lib/iris/tests/unit/util/test__slice_data_with_keys.py

```diff
@@ -105,7 +105,13 @@ def test_1d_all(self):
     def test_1d_tuple(self):
         # The call makes tuples into 1-D arrays, and a trailing Ellipsis is
         # added (for the 1-D case only).
-        self.check((3,), Index[(2, 0, 1),], [(np.array([2, 0, 1]), Ellipsis)])
+        self.check(
+            (3,),
+            Index[
+                (2, 0, 1),
+            ],
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?